### PR TITLE
Create donut.md

### PR DESCRIPTION
**Pull request by jorgeschnura**

### CODE REVIEW

**Comment by eiso** [comment]
This PR should also add the channel to [slack_channels.md](https://github.com/src-d/guide/blob/master/communication/slack_channels.md) and link back to this document from there.

For the rest LGTM, it's a really nice initiative and I am looking forward to having the first match.

**Comment by eiso** [comment]
@jorgeschnura I just saw you merged this, instead of making the changes. Are you opening a new PR for them or should I revert?

**Comment by jorgeschnura** [comment]
I made the changes as you suggested on slack_channels and then merged this one

**Comment by eiso** [comment]
So just the tip for the future is to add those commits to the same branch, so they show up here. It also collapses my comments then and can be approved. When doing it from the GitHub web interface, just select the branch to be this one and edit/commit.

2018-03-02-120111_684x951_scrot

File: communication/donut.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Donut</h1>
+
+One potential downside of being a remote company is that sourceres don't have as many opportunities to have sporadic, informal conversations to get to know each other better.<br>
```
Comment (campoy) [inline communication/donut.md]: sourceres?

Comment (jorgeschnura) [reply communication/donut.md]: Should be sourcerers.
sourcerers = source{d} team members

**Pull request merged**